.travis.yml

```
@@ -4,8 +4,6 @@ language: php
 sudo: false
 
 php:
-  - 7.1
-  - 7.2
   - 7.3
```

```suggestion
  - 7.3
  - 7.4
```

Could you actually test it on PHP 7.4?